draft-ietf-scone-protocol.md

```diff
@@ -528,6 +528,32 @@ a processing error at network elements. For example, they might pick connection
 identifiers of arbitrary length. Network elements can mitigate these attacks
 with implementations that fully conform to the specification of {{packet}}.
 
+## Damage to Other Protocols
+
+Network elements that update SCONE packet fields might do that for datagrams
+exchanged in other protocols.
+This could result in damage to those protocols.
+
+The most serious damage occurs when every datagram is modified,
+because that could mean that the protocol is
+effectively unable to operate end-to-end.
+
+To that end, network elements MUST only update the content of datagrams
+on a given address tuple
+more than a few times each enforcement period.
+Network elements MAY update more often
+immediately after a change in their throughput advice,
+to reduce the reaction time from senders.
+
+In addition, some heuristics might be used
+to detect SCONE-compatible QUIC flows.
+This includes identification of a QUIC handshake on the flow,
```

Comment:
Considering that SCONE is meant to work with any version of QUIC, I'm not sure if we would want to talk about identifying handshakes, as that is a concept specific to version 1 of QUIC.

Reply:
I think that this is fine in the context of applying heuristics.